### DISCUSSION TITLE
v2.0 of Nvchad

### DISCUSSION THREAD

**Discussion by dwskme** [NvChad/NvChad#651] [General]
I heard v2.0 was coming... Is it true? If yes when is it coming? I wanna know what new and improved features are we getting. Please tell me its true....

**Comment by siduck** [comment]
> I heard v2.0 was coming... Is it true? If yes when is it coming? I wanna know what new and improved features are we getting. Please tell me its true....

probably before next year  ,check the 2.0 milestone and contribute if you can

**Reply by dwskme** [reply]
where can i check 2.0 milestone...

**Reply by suciptoid** [reply]
@dwskme https://github.com/NvChad/NvChad/milestone/2

**Reply by siduck** [reply]
@dwskme  @suciptoid  I know I am late, v2.0 will release by tomorrow or the day after it

**Reply by suciptoid** [reply]
awesome @siduck 🚀

**Reply by dwskme** [reply]
finally my wait is over... good work @siduck
